File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_tstvalrn.py
"""Reference emit for TSTVALRN — TSTVAL00 FUNC-PERF USING path → 31."""
# EXPECTED: 31
FUNC = "PERF"
RC = {"INIT": 0, "FUNC": 11, "INTG": 21, "PERF": 31, "ERR": 41}.get(FUNC, 99)
print(f"{RC:02d}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_tstvalrn.java
// EXPECTED: 31
// Reference emit for TSTVALRN — TSTVAL00 FUNC-PERF USING path.
import java.util.Map;
public final class TstvalrnRef {
  public static void main(String[] args) {
    String func = "PERF";
    int rc = Map.of("INIT", 0, "FUNC", 11, "INTG", 21, "PERF", 31, "ERR", 41)
        .getOrDefault(func, 99);
    System.out.printf("%02d%n", rc);
  }
}
